test/libsolidity/semanticTests/array/copying/function_type_array_to_storage.sol

```diff
@@ -46,7 +46,7 @@ contract C {
 }
 // ----
 // test() -> 0x20, 0x14, "[a called][b called]"
-// gas irOptimized: 116530
+// gas irOptimized: 116531
```

Comment:
Also here: it's just one gas difference, which doesn't matter too much, but it's a bit strange that this has an effect at all - the only thing that should be possible as effect of this change should be to combine more equivalent functions, for which an increase of 1 gas is a weird effect... did you look at the diff of optimized Yul here?

Reply:
it's a few `sstore(_1, "")` being replaced by `sstore(_1, _1)`

Reply:
Ok, that's probably a ``DUP`` vs a ``PUSH0``... might even change back with https://github.com/ethereum/solidity/pull/14117. I'm assuming ``_1`` has zero-value :-D - then it's fine.

Reply:
it does!